lib/registry.js

```diff
@@ -223,6 +223,10 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
       'must be an object', modelName);
   }
 
+  if (typeof config.methods === 'object' && config.methods !== null) {
+    settings.methods = config.methods;
```

Comment:
IIUC, this allows developers to modify remoting metadata in `server/model-config.json`. However, this code is called after `_defineRemoteMethods`, therefore I think it won't work in this case:

```js
var BaseModel = loopback.createModel('BaseModel');
var MyCustomModel = loopback.createModel('MyCustomModel');

loopback.configureModel(BaseModel, {
  dataSource: null,
  methods: {
    greet: {
      http: { path: '/greet' },
    },
  },
});

loopback.configureModel(MyCustomModel, {
  dataSource: null,
  methods: {
    greet: {
      http: { path: '/greet' },
    },
  },
});

// I would expect MyCustomModel to have both "greet" and "hello" methods,
// but I suspect MyCustomModel won't have any remote methods
```